### ISSUE TITLE
Script for timing the models

### ISSUE THREAD

**Issue by mehtadushy** [stigma0617/VoVNet.pytorch#2]
Hi

How do you time the models, and what batch size and input image size do you use ?
When I tried to time various models with 224x224 input on different GPUs, the timings I see differ vastly from those reported in the repo here.
TitanXp: Res50(15.5ms B1, 24.4ms B16), VoVNet39(13ms B1, 24.5ms B16)
TitanX: Res50(15.2ms B1, 42.2ms B16), VoVNet39(12.6ms B1, 43.9ms B16)

**Comment by stigma0617** [comment]
@mehtadushy hi,

I have a plan to upload inference script and more efficient models such as VoVNet-27-slim and VoVNet-27-slim-depthwise. 

I measure the inference time by using imageNet validation set (50000 images) with 1 batch size (50000 iterations) and 224x224 input size.

**Comment by mehtadushy** [comment]
Do you measure the time with model.eval() and with torch.no_grad() ?

**Comment by stigma0617** [comment]
@mehtadushy 
of course.

**Comment by mehtadushy** [comment]
I added timing comparisons to VoVNet-39 for different image resolutions with SelecSLS here https://github.com/mehtadushy/SelecSLS-Pytorch
I have also checked in my timing evaluation script
